src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,147 @@
+package core.basesyntax.service;
+
+import static core.basesyntax.service.RegistrationServiceImpl.MINIMAL_USER_AGE;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public static final String DEFAULT_LOGIN = "someLogin";
+    public static final String DEFAULT_PASSWORD = "somePassword";
+    public static final Long DEFAULT_ID = 0L;
+
+    private static StorageDao storageDao;
+    private static RegistrationService service;
+
+    @BeforeAll
+    static void setUp() {
+        storageDao = new StorageDaoImpl();
+        service = new RegistrationServiceImpl(storageDao);
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_notNull_ok() {

```

Comment:
also in ok cases check storage, size should increase and assertEquals for register() return value